# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/PeerConnectionMedia.h
# Commit: 1af1af4020a2
# Full Hash: 1af1af4020a25da7b80c97ddb7816542bcca28ea
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 3
# Description:
#   Bug 1654112 - Create Call instances using the new Call config. r=ng,bwc
#   
#   This sets up our code for creating Call instances so that it works. A number of
#   things are left to do. Most notable fixing the threading model.
#   
# ==============================================================================

diff -r 23b8f56eec4b -r 1af1af4020a2 dom/media/webrtc/jsapi/PeerConnectionMedia.h
--- a/dom/media/webrtc/jsapi/PeerConnectionMedia.h	Tue Feb 23 19:31:35 2021 +0000
+++ b/dom/media/webrtc/jsapi/PeerConnectionMedia.h	Mon Jan 18 15:46:57 2021 +0100
@@ -91,6 +91,8 @@
   // TODO: Let's move the TransceiverImpl stuff to PeerConnectionImpl.
   nsresult AddTransceiver(JsepTransceiver* aJsepTransceiver,
                           dom::MediaStreamTrack* aSendTrack,
+                          SharedWebrtcState* aSharedWebrtcState,
+                          webrtc::WebRtcKeyValueConfig* aTrials,
                           RefPtr<TransceiverImpl>* aTransceiverImpl);
 
   void GetTransmitPipelinesMatching(